ir/ir.def

```diff
@@ -452,14 +452,16 @@ class BlockStatement : Statement, ISimpleNamespace, IAnnotated {
     Annotations getAnnotations() const override { return annotations; }
 }
 
-class MethodCallStatement : Statement {
+class MethodCallStatement : Statement, IAnnotated {
```

Comment:
The P4-16 spec does not allow annotations on MethodCallStatements.
You will not be able to parse a program that has them.